packages/e2e-tests/plugins/disable-animations.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Test Plugin, Disables the CSS animations
+ * Plugin URI: https://github.com/WordPress/gutenberg
+ * Author: Gutenberg Team
+ *
+ * @package gutenberg-test-disable-animations
+ */
+
+/**
+ * Enqueue CSS stylesheet disabling animations.
+ */
+function enqueue_disable_animations_stylesheet() {
+	$custom_css = '* { animation-duration: 1ms !important; }';
```

Comment:
That's aggressive but I like it :)

Reply:
Out of curiosity, why does this have to be `1ms` and not `0`? It seems we leave open the potential for race conditions.

Reply:
For me if we do `0` it means we disable animations entirely and `1ms` is saying the animation are there (to match prod) but are very fast. Not sure if it's a great argument though :P

Reply:
That seems to me like the worst of both worlds though, we're not really testing the animation nor are we disabling them entirely.

Reply:
See #13779. `0ms` seems to be even better in my testing.